src/widgets/Survey/model/hooks/useSurveyState.ts

```diff
@@ -1,16 +1,17 @@
 import { useMemo } from 'react';
 
 import { appletModel } from '~/entities/applet';
+import { PassSurveyModel } from '~/features/PassSurvey';
 
 export const useSurveyState = (activityProgress: appletModel.ActivityProgress) => {
   const items = useMemo(() => activityProgress?.items ?? [], [activityProgress.items]);
 
   const visibleItems = items.filter((x) => !x.isHidden);
 
-  const processedItems = appletModel.conditionalLogicBuilder.process(visibleItems);
+  const processedItems = PassSurveyModel.conditionalLogicBuilder.process(visibleItems);
 
   const conditionallyHiddenItemIds =
-    appletModel.conditionalLogicBuilder.getConditionallyHiddenItemIds();
+    PassSurveyModel.conditionalLogicBuilder.getConditionallyHiddenItemIds();
```

Comment:
I believe, at this point, we don't even need a separate method to return hidden items. We could either:
1) just figure out the hidden items right here, in this function, as we have all items (visibleItems) and  processedItems.
2) or make the `process` method return both hidden and visible items:
```
const availableItems = items.filter((x) => !x.isHidden);
const { visibleItems, hiddenItems } = PassSurveyModel.conditionalLogicBuilder.process(availableItems);
```

Reply:
Done